src/main/java/seedu/address/logic/commands/SettleCommand.java

```diff
@@ -0,0 +1,127 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_AMOUNT;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_STUDENTS;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.student.OwedAmount;
+import seedu.address.model.student.PaidAmount;
+import seedu.address.model.student.Student;
+
+
+/**
+ * Updates owedAmount and paidAmount of a student in the address book by provided amount.
+ */
+public class SettleCommand extends Command {
+
+    public static final String COMMAND_WORD = "settle";
+    public static final String COMMAND_WORD_RANDOM_CASE = "sEttLe";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Updates owed amount and paid amount of the student "
+            + "identified by the index number used in the displayed Student list.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + "[" + PREFIX_AMOUNT + "AMOUNT] ";
```

Comment:
I though square brackets mean optional parameters?

Reply:
You are right. Sorry about that. I have fixed the error.